loops-arrays-objects.js

```diff
@@ -32,9 +32,24 @@ const scores = [
 // D
 // ==========================================
 
-
-
-
+for (let i = 0; i < scores.length; i++) {
+    let score = scores[i].score
+    if (score < 60) {
+        console.log('F')
+    }
+    else if (score >= 60 && score < 70) {
```

Comment:
je zou hier de eerste boolean weg kunnen laten( score >= 60. Deze check je in principe al in de vorige if. geld ook voor de andere else if's